src/main/webapp/i18n/en/clubFamily.json

```diff
@@ -7,6 +7,12 @@
         "createOrEditLabel": "Create or edit a Club Family",
         "notFound": "No CC Family found"
       },
+      "member": {
+        "title": "CClub Family Member",
+        "createLabel": "Add a new Club Family Member",
+        "createOrEditLabel": "Add or edit a Club Family Member",
```

Comment:
UI display name use `CC Family Member` maybe

Reply:
Resolve in dfc7a4da3bb5aa771571d1c94115bf52c2f87f19